Update E06

purr-diesel opened the pull request.

purr-diesel committed:
Update E06

doctortheisen reviewed (changes requested):
Remove inline comment. These are production templates for non-coders.

doctortheisen committed:
Update E06

Removed a comment that would have appeared on the front end for production content.
